styles/_mixins.scss

```diff
@@ -22,3 +22,25 @@
   pointer-events: none;
   opacity: 0.2;
 }
+
+// todo: refactor this style once we have designs
+@mixin processingOverlay {
+  position: relative;
```

Comment:
The overlay doesn't always cover the element it's overlaying, height: 100% might fix that.

![image](https://cloud.githubusercontent.com/assets/1584275/17898685/b6ef5870-6926-11e6-8d14-2839f0561d1e.png)